src/components/FileMessage.tsx

```diff
@@ -0,0 +1,59 @@
+import { FileMessage as ChatFileMessage } from '@sendbird/chat/message';
+import styled from 'styled-components';
+import { useGroupChannelContext } from '@sendbird/uikit-react/GroupChannel/context';
+// import { FileViewerComponent } from '@sendbird/uikit-react/ui/FileViewer';
+// import {downloadFileWithUrl, noop} from '../utils';
+// import {createPortal} from 'react-dom';
+
+const Root = styled.div`
+  width: 100%;
+`;
+
+const Image = styled.img`
+  border-radius: 16px;
+  width: 100%;
+`;
```

Comment:
It seems like these simple components don't need to use styled components.
Since styled components are executed at runtime, they need to be executed, and template literals not to be minified, making the code larger.

How about just using inline styles on regular tags?

Reply:
> Since styled components are executed at runtime, they need to be executed, and template literals not to be minified, making the code larger.

Thats a good point. Actually I learned that using styled component has no css class identifier which has two crucial cons:
- Hard to locate source code from developer tools.
- Customers cannot override their own style.

I think we should remove styled component later. For the same reason, I want to avoid using inline styling. Also inline styling has high precedence so customer may complain later. How about using scss files just like UIKit?

Reply:
Agreed. However, the highest priority in the current widget is to improve performance and reduce bundle size.

Reply:
Later on, it would be good to transition to zero-runtime tools like vanilla extract.

- https://linaria.dev/
- https://stylexjs.com/
- https://vanilla-extract.style/

Reply:
https://youtu.be/9JZHodNR184?si=tw6eOb_pxqsHyEt9&t=220